### DISCUSSION TITLE
RF Communication Supporting Synoxo SYN590R RF?

### DISCUSSION THREAD

**Discussion by RaW79** [arendst/Tasmota#20991] [Q&A] [answered]
Hi
tested on Tuya TWYD 4ch-RF board (CB3S Tuya Chip) to be replaced by ESP8266 12F operating. 

After some smaller GPIO PIN assignment issues which could be fixed by isolation or additional bridges the board is working for all relays and buttons with Tasmota.  

![1710250308656-641529b7-71d1-4e13-bc7f-9f3f6bc3061c-grafik](https://github.com/arendst/Tasmota/assets/16465702/dfd059ef-7e4b-479d-939c-318111a52d14)

But I´m struggeling to make use of the 433MHz RF function. The chip is connected to GPIO 10 and i used self compiled Tasmota version including RF tranceiver package assigning RFrecv to GPIO10. In addition set the rfprotocol to a -> all but without any results in the console. The identified RF chip seems to be a Synoxo SYN590R RF which seems to be similar to the documented and supported SRX882.

Do you have any experience or feedback in using the SYN590R with the rf communication package in Tasmota?

Thx!

**Comment by Jason2866** [comment, marked as answer]
Check you hardware setup and connection. The SYN590R is working with RCswitch/Tasmota.
Have you used a firmware with RCswitch support and configured the GPIO.

**Reply by sfromis** [reply]
Also verify that your soldering is fully working. If you config the pin from the RF chip as a "Counter", you should see the number increment with activity.

**Reply by RaW79** [reply]
Thank you both very much. Your confirmation in the support of SYN590R helped to do some additional tests and another GPIO showed me that Tasmota is working fine with RF function as expected. Took some time but then identified a connection issue because of the soldering pad of the pcb at GPIO 10. After fixing it, everthing works. Great, Thanks!

**Reply by sfromis** [reply]
Noticed that there seems to be a lot of flux left around the soldering. That's fine to help soldering, but it is recommended to clean it afterwards (maybe using some IPA), as flux is often acidic, which can lead to corrosion.
